grakn-test/src/test/java/ai/grakn/client/BatchMutatorClientTest.java

```diff
@@ -73,6 +75,7 @@ public void setupSession(){
     }
 
     @Test
+    @Ignore("Testing log output, it's pretty flimsy. It's also prone to race conditions (the whole test task halts forever)")
```

Comment:
Yes, this is 100% true. I think I've identified the race condition that causes it to randomly fail, but could not figure out how to prevent it :P